src/core/column-type/column.model.js

```diff
@@ -10,7 +10,7 @@ export class ColumnModel {
 		this.pin = null;
 		this.origin = 'specific';
 		this.source = 'user';
-		this.class = 'data';
+		this.class = '';
```

Comment:
I didn't see direct problems.
User can define both data and control columns, so in my opinion he has to define class himself when he need.